### PR TITLE
SEO: robots.txt and sitemap.xml

### LINKED ISSUE #38: Sitemap
Add sitemap

### PR DESCRIPTION
**Pull request by Jordan141**
This PR will deal with the SEO issues of robots.txt and sitemap.xml setup

Resolves #38 , #39

### CODE REVIEW

**Comment by michaelKurowski** [comment]
Could you update read me with instruction of things to do when ones put article blog into production?

**Comment by michaelKurowski** [comment]
Can we autogenerate sitemap, instead of requiring manual change? It's easy to miss stuff when a project requires manual maintenance of things like that.

File: app/public/robots.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+User-Agent: *
```
Comment (michaelKurowski) [inline app/public/robots.txt:1]: Disallow also user profiles

Comment (michaelKurowski) [reply app/public/robots.txt:1]: Nvm, we agreed they're private

**Comment by Jordan141** [comment]
> Can we autogenerate sitemap, instead of requiring manual change? It's easy to miss stuff when a project requires manual maintenance of things like that.

We can, this will be added dynamically once the frontend and routes changes are finished,

**Pull request merged**